src/js/components/charts/chartDeath.js

```diff
@@ -0,0 +1,33 @@
+import Data from '../data/Data';
+
+export default class chartDeath {
+    constructor() {
+        this.globalInfo2 = null;
+    }
+
+    async create() {
+        const data = new Data();
+        await data.initData()
+            .then(() => {
+                this.globalInfo2 = data.getGlobalPeriodData();
+            });
+        const ctx = document.getElementById('myChartDeath').getContext('2d');
+        const chart = new Chart(ctx, {
+            type: 'line',
+
+            data: {
+                labels: ['Сентябрь', 'Октябрь', 'Ноябрь'],

```

Comment:
I think that this values should be a single constant because you used it in more than 1 file